unit-tests/src/main/scala/tests/Main.scala

```diff
@@ -2,6 +2,8 @@ package tests
 
 import java.lang.System.exit
 
+import scala.scalanative.SyscallsSuite
```

Comment:
This change is not necessary, it doesn't have any effect. Test suites are resolved automatically.